### ISSUE TITLE
Modification

### ISSUE THREAD

**Issue by Schadenfreude-star** [kianData/PyTorch-Multivariate-LSTM#1]
out_cols_max = torch.Tensor([[max(df[out_cols[j]])]])
out_cols_min = torch.Tensor([[min(df[out_cols[j]])]])
y_train_pred = (y_train_pred * (out_cols_max - out_cols_min) + out_cols_max).detach().numpy()
y_train = (y_train * (out_cols_max - out_cols_min) + out_cols_max).detach().numpy()
y_test_pred = (y_test_pred * (out_cols_max - out_cols_min) + out_cols_max).detach().numpy()
y_test = (y_test * (out_cols_max - out_cols_min) + out_cols_max).detach().numpy()

I change your codes for **invert prediction**  parts into above codes because some tensor shape issues, just for references.

**Comment by KianML** [comment]
No worries, thanks for your feedback. I’ll try to fix it.
